docs/quickstart/quickstart-non-docker.rst

```diff
@@ -0,0 +1,136 @@
+.. _ksql_quickstart_non_docker:
+
+Running KSQL Locally
+====================
+
+This topic describes how to setup a Kafka cluster and start KSQL in your local environment. After you complete these steps,
+you can return to the :ref:`create-a-stream-and-table` and start querying the data in the Kafka cluster.
+
+.. contents:: Contents
+    :local:
+    :depth: 1
+
+Prerequisites
+    - :ref:`Confluent Platform 3.3.0 <installation>` or later is installed. This installation includes a Kafka broker, ZooKeeper, Schema Registry, REST Proxy, and Kafka Connect.
+    - If you installed Confluent Platform via TAR or ZIP, navigate into the installation directory. The paths and commands used throughout this quick start assume that your are in this installation directory.
+    - `Maven <https://maven.apache.org/install.html>`__
+    - `Git <https://git-scm.com/downloads>`__
+    - Java: Minimum version 1.8. Install Oracle Java JRE or JDK >= 1.8 on your local machine
+
+Start Kafka
+-----------
+
+Navigate to the ``confluent-3.3.0`` directory and start the Confluent
+Platform using the new Confluent CLI (part of the free Confluent Open
+Source distribution). ZooKeeper is listening on ``localhost:2181``,
+Kafka broker is listening on ``localhost:9092``, and Confluent Schema
+Registry is listening on ``localhost:8081``.
+
+.. code:: bash
+
+    $ ./bin/confluent start
```

Comment:
We're not consistent IMHO.  Sometimes we say `./bin/confluent start`, sometimes we say `<path-to-confluent-3.3.0>/bin/confluent start`.  Can we pick one and stick with it?  Note that this should be consistent across the docs of all CP components.

Reply:
should be `<path-to-confluent>/bin/confluent start`.